spec/css/media/bubbling.hrx

```diff
@@ -23,3 +23,31 @@
     }
   }
 }
+
+<===>
+================================================================================
+<===> preserve_merge_after_bubble/input.scss
```

Comment:
Doesn't the change for `equalsIgnoringChildren` also affect other nodes? I wonder if it would make sense to add tests for other "container" nodes where bubbling may be affected by the change 🤔

Reply:
I'm not sure there are actually other situations that trigger this behavior. Nested selectors always get pushed below declarations, and other at-rules aren't merged with their parent and so just end up normally nested rather than bubbled.